controls/V-93509.rb

```diff
@@ -22,8 +22,8 @@
     dsquery * \"cn=Default Query Policy,cn=Query-Policies,cn=Directory Service, cn=Windows NT,cn=Services,cn=Configuration,dc=[forest-name]\" -attr LDAPAdminLimits
 
     The quotes are required and dc=[forest-name] is the fully qualified LDAP name of the domain being reviewed (e.g., dc=disaost,dc=mil).
-    If the results do not specify a \"MaxConnIdleTime\" or it has a value greater than \"#{input('maximum_idle_time')}\" (#{input('maximum_idle_time_phrase')}), this is a finding."
-  desc  'fix', "Configure the directory service to terminate LDAP-based network connections to the directory server after #{input('maximum_idle_time_phrase')} of inactivity.
+    If the results do not specify a \"MaxConnIdleTime\" or it has a value greater than \"#{input('maximum_idle_time')/60}\" minutes, this is a finding."
+  desc  'fix', "Configure the directory service to terminate LDAP-based network connections to the directory server after #{input('maximum_idle_time')/60} minutes of inactivity.
```

Comment:
The fix text looks good, but the check text needs the input listed twice. Here's a screenshot of the diff for reference:

![Screen Shot 2023-05-10 at 3 13 21 PM](https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/assets/11844975/e686b880-3539-49e0-98df-e22dd18f8427)